eng/pipelines/linux.yml

```diff
@@ -85,10 +103,12 @@ jobs:
         - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
           - linuxDefaultQueues: Centos.7.Amd64.Open+RedHat.7.Amd64.Open+Debian.8.Amd64.Open+Ubuntu.1604.Amd64.Open+Ubuntu.1804.Amd64.Open+OpenSuse.42.Amd64.Open+Fedora.27.Amd64.Open
           - linuxArm64Queues: Ubuntu.1604.Arm64.Open
+          - linuxArmQueues: '\(Debian.9-arm32\)Ubuntu.1604.Arm32.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm32v7-0a0ebdd-20190312215452'
       
         - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
           - linuxDefaultQueues: Centos.7.Amd64+RedHat.7.Amd64+Debian.8.Amd64+Debian.9.Amd64+Ubuntu.1604.Amd64+Ubuntu.1804.Amd64+Ubuntu.1810.Amd64+OpenSuse.42.Amd64+SLES.12.Amd64+SLES.15.Amd64+Fedora.27.Amd64+Fedora.28.Amd64
           - linuxArm64Queues: Ubuntu.1604.Arm64
+          - linuxArmQueues: '\\(Debian.9-arm32\\)Ubuntu.1604.Arm32@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm32v7-0a0ebdd-20190312215452'
```

Comment:
Why do we need to use ' and escape parenthesis? Is `(` not allowed?

Reply:
Maybe the reason why it is not showwing correctly is because of the escaping.

Reply:
this is work in progress. As the '(' is special character passed to shell scripts. 
Also https://github.com/dotnet/arcade/issues/2261 needs to go in.